src/main/java/filter/reader/WordReader.java

```diff
@@ -0,0 +1,77 @@
+package filter.reader;
+
+import filter.utils.Encoder;
+import filter.writer.WordWriter;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.util.*;
+
+
+public class WordReader {
+    private static final String DATA = "data.txt";
+    private static final String VOCABULARY = "vocabulary.txt";
+    private static final String CHARACTERS = ",|!|\\(|\\)|\\.";
+    private static final String REPLACEMENT = " ";
+    private static final String REGEX = "\\s+";
+    private static final int NUM = 20;
```

Comment:
just DATA ??
poor description for all constants

ex. DATA_FILE, EXTRA_CHARS, etc.